SDK/src/NDK/Widgets/TextAreaWidget.cpp

```diff
@@ -288,6 +289,28 @@ namespace Ndk
 				return true;
 			}
 
+			case Nz::Keyboard::Tab:
+			{
+				if (!m_tabEnabled)
+					return false;
+
+				if (HasSelection())
```

Comment:
This is not the correct way of handling selections: when a text is selected it should append/remove a tab character at the start of the line.